examples/time_frequency/plot_compute_raw_data_spectrum.py

```diff
@@ -70,3 +71,18 @@
 
 ax.set_title('Four left-temporal magnetometers')
 plt.legend(['Without SSP', 'With SSP', 'SSP + Notch'])
+
+# Alternatively, you may also create PSDs from Raw objects with psd_XXX
+f, ax = plt.subplots()
+psds, freqs = psd_multitaper(raw, low_bias=True, tmin=tmin, tmax=tmax,
```

Comment:
been looking into this a bit - do the projections mix information between the channels?

In the example we're using a subset of picks for plotting: `picks = picks[:4]`. However, this is also passed to the notch_filter function as well: `raw.notch_filter(np.arange(60, 241, 60), picks=picks, n_jobs=1)`.

Could it be that we're notching these 4 channels, and then mixing in information from un-notched channels? I notice that if instead I do:

```python
picks_notch = mne.pick_types(raw.info, meg=True, eeg=True)
raw.notch_filter(np.arange(60, 241, 60), picks=picks_notch, n_jobs=1)
raw.plot_psd(tmin=tmin, tmax=tmax, fmin=fmin, fmax=fmax, n_fft=n_fft,
             n_jobs=1, proj=True, ax=ax, color=(1, 0, 0), picks=picks)
```
Then it works...